Amigolden/src/app/components/map/map.component.ts

```diff
@@ -88,19 +106,28 @@ export class MapComponent implements OnInit {
     this.locationSelected.emit({ location: this.selectedLocation });
   }
 
-  cancel() {
-  }
+  cancel() {}
 
   ngOnInit() {
     // load Places Autocomplete
+    this.searchForm = this.fb.group({
+      autocomplete_input: [""],
+    });
     this.mapsAPILoader.load().then(() => {
       this.setCurrentLocation();
       this.geoCoder = new google.maps.Geocoder();
+      let inputfield = document
```

Comment:
this file shouldn't have changes but for future reference, you don't want to grab elements the native way because it circumvents angulars life cycle, and could cause issues with the DOM,   see how I was grabbing the element prior:
this.searchElementRef.nativeElement